src/App.vue

```diff
@@ -69,4 +69,15 @@ export default {
     min-height: 100%;
   }
 }
+html {
+  scroll-padding-top: 80px;
+}
+div,section {
+  scroll-margin-top: 80px;
+}
```

Comment:
yeah i was trying that when click on link the content gets behind the navbar because its fixed so i was trying putting this css but it did not worked and forgot to remove.